### PR TITLE
Adds more ways to ignite e-cigarettes.

### PR DESCRIPTION
**Pull request by Sadhorizon**
## About The Pull Request
Do you want your character to vape, but did they not spawn with a PDA? Well, worry not, this PR adds new ways to ignite your e-cigarettes/e-cigars!

First one, is with a tablet. Works essentially the same as with a PDA. There is way more jobs that get a free tablet than those that get a PDA. Plus, you can get those from loadout.

Second one, is with an APC. Smash your vape on the APC and it will get ignited! How does it work? Clearly designers were just very thoughtful and included a vape igniter or whatever you use!

## Why It's Good For The Game
Items being actually usable is nice.

## Changelog

:cl:
tweak: E-cigarettes and e-cigars can now be turned on with tablets and APCs.
/:cl:

### CODE REVIEW

**Comment by Sweettoothart** [comment]
lets GOOOOOO HELL YEAH

**Comment by KittyNoodle** [comment]
make it so you can light them on elzuosa

**Comment by Sadhorizon** [comment]
> make it so you can light them on elzuosa

Mmm, not really a fan of that idea.

Feel free to open a follow-up PR if you want to see that happen though.

File: code/game/objects/items/cigs_lighters.dm [hunk 1]
```diff
@@ -920,6 +920,30 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 		user.update_inv_wear_mask()
 		user.update_inv_hands() //EndWS edit - Lightable e-cigarettes
```
Comment (thgvr) [inline code/game/objects/items/cigs_lighters.dm]: Could you remove these WS edit comments? Good practice to nuke them when you see them

**Pull request merged**